tribits/core/utils/ParseVariableArguments.cmake

```diff
@@ -248,12 +248,12 @@ endmacro()
 
 # NOTE: If the above function turns out to be a performance bottle neck, there
 # are a few things that could be done to improve performance.  One thing you
-# could do is repalce the o(len(arg_names)) and o(len(option_names)) lookups
+# could do is replace the o(len(arg_names)) and o(len(option_names)) lookup
```

Comment:
Technically I think the plural "lookups" is correct because there are two lookups being reference here but okay.